### PR TITLE
[Debugger] Expression Language - comparison operators

### PR DESCRIPTION
**Pull request by shurivich**
## Motivation
Test expression language comparison operators

### CODE REVIEW

File: tests/debugger/probes/expression_language_comparison_operators.json
Comment (dudikeleti) [inline tests/debugger/probes/expression_language_comparison_operators.json:1]: Do we want to test also `!` or `not` ?
Also I'm not sure if this is supported in the final version of the RFC, but I see that we have two version of each equality operator, for example `eq` => `--`, `gt` => `>` etc etc. Do we want to test it as well?

**Review by dudikeleti — APPROVED** [review]
LGTM. Left a comment.

**Review by jpbempel — APPROVED** [review]
LGTM
but we are testing only integers here.
What is missing:
- floats
- strings
- objects, `null`

File: tests/debugger/probes/expression_language_comparison_operators.json
Comment (shurivich) [reply tests/debugger/probes/expression_language_comparison_operators.json:1]: Good point! If we support it - we should. 
I will add it to upcoming PRs.

**Review by robertomonteromiguel — APPROVED** [review]

**Pull request merged**